### PR TITLE
Data update - November 2019

### PR DESCRIPTION
**Pull request by MattCowgill**
This is a data update. 

Note: I made some minor changes to `put-data.R`. These primarily involved coercing objects to `data.frame()` to avoid the warnings that the latest dtplyr throws when using dplyr syntax on data.table objects. 

I ran `pkgdown::build_site()` and encountered an error related to `budget-2019.Rmd`. There's an error at line 729:
```
Quitting from lines 729-743 (budget-2018.Rmd) 
Error in eval(jsub, SDenv, parent.frame()) : object 'Ind' not found
```

`R CMD check` passes with no errors, warnings or notes locally and on Travis.

### CODE REVIEW

File: data-raw/Age-Pension-assets-test-1997-present.tsv [hunk 1]
```diff
@@ -113,7 +117,7 @@ TRUE	TRUE	FALSE	2015-07-01	440500
 TRUE	TRUE	FALSE	2016-07-01	448000
 TRUE	TRUE	FALSE	2017-01-01	575000
 TRUE	TRUE	FALSE	2017-07-01	583500
-TRUE	TRUE	FALSE	2018-07-01	594500
+TRUE	TRUE	FALSE	2019-07-01	605000
```
Comment (DaniWood) [inline data-raw/Age-Pension-assets-test-1997-present.tsv]: Is this mean to be an insertion, rather than a replacement?

Comment (MattCowgill) [reply data-raw/Age-Pension-assets-test-1997-present.tsv]: oh god, yes it is. I'll fix.

Comment (DaniWood) [reply data-raw/Age-Pension-assets-test-1997-present.tsv]: I suspect it's a problem with the logic with `Age_pension_assets_test_by_year <- ` where it uses the original data regardless of its age. The original logic was due to the the `      htmltab::htmltab("http://guides.dss.gov.au/guide-social-security-law/4/10/3", which = 10)` being too flaky to run every time.

Comment (MattCowgill) [reply data-raw/Age-Pension-assets-test-1997-present.tsv]: fixed now

File: data-raw/put-data.R [hunk 2]
```diff
@@ -966,15 +966,18 @@ Age_pension_assets_test_by_year <-
...
     stopifnot(identical(names(age_pension_assets_test_dss_gov_au),
                         c("Date",
                           "Homeowners >> Single",
                           "Homeowners >> Couple",
-                          "Homeowners >> Illness Separated Couple", 
+                          "Homeowners >> Illness separated couple", 
                           "Non-homeowners >> Single",
                           "Non-homeowners >> Couple",
-                          "Non-homeowners >> Illness Separated Couple", 
+                          "Non-homeowners >> Illness separated couple",
```
Comment (DaniWood) [inline data-raw/put-data.R]: ...

**Comment by DaniWood** [comment]
Appveyor is spurious -- using an old version of a package.

**Pull request merged**